src/clj/harja/kyselyt/paallystys.sql

```diff
@@ -43,7 +43,7 @@ WHERE ypk.urakka = :urakka
   AND (:vuosi :: INTEGER IS NULL OR (cardinality(vuodet) = 0
                                          OR vuodet @> ARRAY [:vuosi] :: INT []))
   AND ypk.poistettu IS NOT TRUE
-  AND ((:paikkauskohteet ::TEXT IS NULL)
+  AND ((:paikkauskohteet ::TEXT IS NULL AND p."yllapitokohde-id" IS NULL)
```

Comment:
Onko tämä tarkoituksella tässä? Näyttää omaan silmään vähän oudolta sen tarkemmin tutkimatta

Reply:
Joo, tässä oli bugi. Lisäämällä tuo IS NULL saadaan mutuaalisesti poissulkeva SQL-kysely, jotta POT-lomakkeet tulevat kälissä vain joko päällystykseen tai paikkaukseen, ei molempiin.